<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>141</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="load_img_btn">
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_video_btn">
       <property name="text">
        <string>Load Video</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>171</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>1. Baclground Subtraction</string>
       </property>
       <widget class="QPushButton" name="BackgroundSubstraction_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1. Baclground Subtraction</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>2. Optical Flow</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>171</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="Preprocess_btn">
           <property name="text">
            <string>2.1 Preprocessing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="video_tracking_btn">
           <property name="text">
            <string>2.2 Video Tracking</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>3. PCA</string>
       </property>
       <widget class="QPushButton" name="PCA_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>3.3 Dimension Reduction</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>661</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>4. MNIST Classifier Using VGG19</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>661</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="sol4_layout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="show_model_btn">
             <property name="text">
              <string>1. Show Model Structure</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_acc_loss_btn">
             <property name="text">
              <string>2. Show Accurancy and Loss</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="predict_mnist_btn">
             <property name="text">
              <string>3. Predict</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reset_btn">
             <property name="text">
              <string>4. Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mnist_predict">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Predict = </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="paint_view"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>5. ResNet50</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>11</y>
          <width>661</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="load_q5Img_btn">
             <property name="text">
              <string>Load image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_q5_img_btn">
             <property name="text">
              <string>5.1 Show Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_resnet50_btn">
             <property name="text">
              <string>5.2 Show Model Structure</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Show_Comprasion_btn">
             <property name="text">
              <string>5.3 Show Comprasion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resnet50_predict_btn">
             <property name="text">
              <string>5.4 Inference</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QGraphicsView" name="input_img"/>
           </item>
           <item>
            <widget class="QLabel" name="resnet_predict">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Predict = </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
